tests/driver_srf04/README.md

```diff
@@ -0,0 +1,14 @@
+tests/driver_srf04
+================
+This example shows the usage of an srf04 module.
+The application exerts a timer and two gpio.
+The module is an ultrasonic range finder with a small protocoll,
+hence an implementation can be kept simple.
+
+Usage
+=====
+
+Build, flash and start the application:
+```
+export BOARD=your_board
+make -j 2 flash term
```

Comment:
Missing closing \`\`\`
Or just indent the block with 4 spaces (and remove the opening \`\`\`). I would put the command line on a single line and remove `-j2`